### PR TITLE
Add support for customizing service profiles in Spinnaker helm chart.

### PR DESCRIPTION
**Pull request by cparedes**
Signed-off-by: Ian Paredes <cp@redbluemagenta.com>

#### What this PR does / why we need it:

This adds support for customizing service profiles for Spinnaker. This is necessary to enable certain features such as pipeline permissions, as well as allowing people to workaround issues with fixes like this: https://github.com/spinnaker/spinnaker/issues/1630#issuecomment-402712481

#### Which issue this PR fixes

N/A

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [ ] Variables are documented in the README.md

### CODE REVIEW

File: stable/spinnaker/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Open source, multi-cloud continuous delivery platform for releasing software changes with high velocity and confidence.
 name: spinnaker
-version: 1.2.0
+version: 1.2.1
```
Comment (cpanato) [inline stable/spinnaker/Chart.yaml]: adding new functionality should but the minor
```suggestion
version: 1.3.0
```

**Comment by cparedes** [comment]
rebased

**Comment by paulczar** [comment]
/lgtm

Perfect!  I was about to make a very similar PR!  thanks for saving me the effort :)

**Pull request merged**